components/services/bat_ledger/bat_ledger_impl.cc

```diff
@@ -184,9 +184,25 @@ void BatLedgerImpl::FetchGrants(const std::string& lang,
   ledger_->FetchGrants(lang, payment_id);
 }
 
+
+// static
+void BatLedgerImpl::OnGetGrantCaptcha(
+    CallbackHolder<GetGrantCaptchaCallback>* holder,
+    std::string image,
+    std::string hint) {
+  if (holder->is_valid())
```

Comment:
Let's confirm that holder isn't null before accessing it here.

Reply:
added `DCHECK(holder)`